src/world_interface/real_world_interface.cpp

```diff
@@ -28,17 +29,86 @@ struct timeval last_odom_reading_;
 transform_t last_odom_tf_;
 double odom_dtheta_, odom_dx_;
 
+// map that associates camera id to the camera object
+std::map<CameraID, std::shared_ptr<cam::Camera>> cameraMap;
+
+void setupCameras() {
+	try {
+		auto arCam = std::make_shared<cam::Camera>();
+		arCam->openFromConfigFile(Constants::AR_CAMERA_CONFIG_PATH);
+		cameraMap[Constants::AR_CAMERA_ID] = arCam;
+	} catch (const std::exception &e) {
+		log(LOG_ERROR, "Error opening camera with id %s:\n%s\n", Constants::AR_CAMERA_ID, e.what());
+	}
```

Comment:
We may want to map camera IDs to their config file paths in the future. Can be saved for a later PR though.

Reply:
That makes sense, it would probably simplify the `setupCameras()` function too, so the only change required to add/remove cameras is to just update the map.